src/main/java/duke/MainWindow.java

```diff
@@ -0,0 +1,90 @@
+package duke;
+
+import javafx.animation.PauseTransition;
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.util.Duration;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+
+    private Parser parser;
+    private Ui ui;
+    private TaskList taskList;
+    private Storage storage;
+
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
+    private String filePath = "../duke/data/duke.txt";
+
+    /**
+     * Loads tasks from file and greets the user.
+     *
+     * @throws FileNotFoundException When the file to store the tasks is not found.
+     */
+    @FXML
+    public void initialize() throws FileNotFoundException {
+        scrollPane.vvalueProperty().bind(dialogContainer.heightProperty());
+
+        ui = new Ui();
+        storage = new Storage(filePath);
+
+        //Load file
+        try {
+            taskList = new TaskList(storage.load(), ui);
+        } catch (DukeException e) {
+            String err = DukeStub.getResponse(ui.showInvalidFile());
+            dialogContainer.getChildren().addAll(
+                    DialogBox.getDukeDialog(err, duke)
+            );
+            taskList = new TaskList(ui);
+        }
+
+        //Greets the user
+        String greet = DukeStub.getResponse(ui.greet());
+        dialogContainer.getChildren().addAll(
+                DialogBox.getDukeDialog(greet, duke)
+        );
+    }
+
+    /**
+     * Creates two dialog boxes, one echoing user input and the other containing Duke's reply and then appends them to
```

Comment:
This comment is verbose; in addition, the method name and the comments don't seem to be in sync.
Can you think of something succinct?